dags/sentinel-2_data_transfer.py

```diff
@@ -154,21 +171,22 @@ def start_transfer(metadata):
     """
 
     s3_hook_oregon = S3Hook(aws_conn_id=US_CONN_ID)
-    s3_filepath = get_self_link(metadata)
+    s3_filepath = get_src_link(metadata)
 
     # Check file exists
     bucket_name, key = s3_hook_oregon.parse_s3_url(s3_filepath)
-    key_exists = s3_hook_oregon.check_for_key(
-        key, bucket_name=SRC_BUCKET_NAME
-    )
+    key_exists = s3_hook_oregon.check_for_key(key, bucket_name=SRC_BUCKET_NAME)
     if not key_exists:
-        raise ValueError(
-            f"{key} does not exist in the {SRC_BUCKET_NAME} bucket"
-        )
-    
+        raise ValueError(f"{key} does not exist in the {SRC_BUCKET_NAME} bucket")
+
     try:
         s3_hook = S3Hook(aws_conn_id=AFRICA_CONN_ID)
-        s3_hook.load_string(string_data=json.dumps(metadata), key=key, bucket_name=DEST_BUCKET_NAME)
+        s3_hook.load_string(
+            string_data=json.dumps(metadata),         
+            replace=True,
+            key=key,
```

Comment:
Replace=True will rewrite the file if it exists.

Reply:
Thinking about this twice, I don't want to replace existing datasets without knowing so disabling it.